src/value.rs

```diff
@@ -1534,10 +1527,10 @@ impl<'a> Set<'a> {
 
 impl<'a> Function<'a> {
     /// Create a function in the current execution context for a given callback.
-    pub fn new(isolate: &'a isolate::Isolate,
+    pub fn new<'b>(isolate: &'a isolate::Isolate,
```

Comment:
Is this lifetime safe? Now we're saying that the closure can live for less time than the isolate, which means that the closure might be dropped in rust land when the user calls it in Javascript land.